src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -396,6 +399,31 @@ AmendmentTableImpl::doValidation (
     return amendments;
 }
 
+std::vector <uint256>
+AmendmentTableImpl::getDesired ()
+{
```

Comment:
Good suggestion.  Minor nit: `doValidation` is now kind of misnamed, and its comment is not quite accurate: 

> [c]alled by the consensus code when we need to add feature entries to a validation